test/standard_api/standard_api_test.rb

```diff
@@ -725,5 +725,19 @@ def normalizers
 
     assert_equal [1], JSON(response.body)
   end
+  
+  test 'preloading polymorphic associations' do
+    p1 = create(:property)
+    p2 = create(:property)
+    a1 = create(:account, subject: p1)
+    a2 = create(:account, subject: p2)      
+    
+    assert_no_sql("SELECT \"properties\".* FROM \"properties\" WHERE \"properties\".\"id\" = $1 LIMIT $2") do
+      get accounts_path(limit: 2, include: :subject, format: 'json')
+      
+      assert_equal p1.id, a1.subject_id
+      assert_equal p2.id, a2.subject_id
+    end
```

Comment:
Lets test multiple models

```suggestion
p1 = create(:property)
    p2 = create(:property)
    c1 = create(:camera)
    a1 = create(:account, subject: p1)
    a2 = create(:account, subject: p2)
    a3 = create(:account, subject: c1)
    
    assert_no_sql("SELECT \"properties\".* FROM \"properties\" WHERE \"properties\".\"id\" = $1 LIMIT $2") do
      get accounts_path(limit: 20, include: :subject, format: 'json')
      
      assert_equal p1.id, a1.subject_id
      assert_equal p2.id, a2.subject_id
      assert_equal c1.id, a3.subject_id
    end
```